core/ai.py

```diff
@@ -146,8 +134,10 @@ def chat(chat_id, input_message):
                         tool_outputs=tool_output_array
                     )
                     run, status = get_run_status(run, client, thread)
-
-                    message = get_assistant_message(client, thread.id)
+                    
+                    resp1 = get_assistant_message(client, thread.id)
+                    message = bhashini_output(resp1, lang=lang)
```

Comment:
Please remove bhashini_output() from chat() function and call it in the bhashini_text_chat() function after calling chat(). In that way bhashini logic and llm chat module will stay separate. We can easily switch from Assistant API to langchain or maybe even open source models.

Reply:
done